src/lib/utilities.js

```diff
@@ -277,3 +277,7 @@ export function filterByEnvironment(development, beta, production) {
   }
   return development;
 }
+
+export const googleClient = '870681894101-ce7tdp6h9fvt2jfrqaaalim1s2n48ie3.apps.googleusercontent.com'; // eslint-disable-line max-len
```

Comment:
Not a fan of this variable name, let's name it `googleClientID` yeah? Unless you can think of something better.

Just another thought actually, where did you get this ID from? Did you create a new Google API account?

Reply:
Yeah I created a new API account.

Reply:
Okay, we‘ll want to consolidate that with our current Google Drive API account, but that's nothing you will have to do (apart from changing the client ID), I'll look into that at some point in the near future and get to you with the new ID

Reply:
okay, I should've asked you about it haha. thanks.